### ISSUE TITLE
FR: Control sizing (like Bootstrap)

### ISSUE THREAD

**Issue by nathanpitman** [harvesthq/chosen#1989]
It would be really nice to have the option of a number of different control sizes (as with Bootstrap). i.e. .input-lg etc.

http://getbootstrap.com/css/#forms-control-sizes

Chosen looks great and would look even more superb in big chunky forms if we could control it's sizing natively. :)

**Comment by phantom7** [comment]
Is there a way to adjust height as well as width? I'm a bit of a Noob. Apologies if I'm asking a silly question as I don't see a "height" option. If there's no clean solution, is there a hack one can suggest?

**Comment by stof** [comment]
@phantom7 chosen does not set any explicit height inline. If you want to control the height, it is a matter of adding some CSS rules

**Comment by philipstratford** [comment]
I agree that it'd be nice if Chosen would inherit Bootstrap sizing classes like **input-sm** and **input-lg**. At the moment I'm getting around the fact that it doesn't by wrapping selects using Chosen inside elements with **form-group form-group-sm**, which works, but it's untidy.
